cvat/apps/engine/backup.py

```diff
@@ -1036,37 +1053,65 @@ def _import_project(filename, user, org_id):
     db_project = project_importer.import_project()
     return db_project.id
 
-def create_backup(db_instance, Exporter, output_path, logger, cache_ttl):
+
+def create_backup(
+    # FUTURE-FIXME: there db_instance_id should be passed
+    db_instance: models.Project | models.Task,
```

Comment:
Why not change this parameter if the function args are already changed in this PR?